action.yml

```diff
@@ -1,8 +1,8 @@
-name: Nix Installer
+name: Determinate Nix Installer
```

Comment:
Think it's `The Determinate Nix Installer` ya?

Reply:
```suggestion
name: The Determinate Nix Installer
```